File: week3 - Application Development 1 CRUD with Cassandra/crud-python/delete.py
#!/usr/bin/env python3
from db_connection import Connection

# this is a delete statement in python
try:
    connection = Connection()
    output = connection.session.execute(
        "DELETE FROM killrvideo.user_credentials WHERE email = %s",
        ['cv@datastax.com']
    )
except:
    print('Failure')
else:
    print('Success')
finally:
    connection.close()

File: week3-AppDev-crud/crud-nodejs/delete.js
const connection = require('./db_connection')

// this is a delete statement in nodejs
connection.client.execute(
    'DELETE FROM killrvideo.user_credentials WHERE email = ?',
    ['cv@datastax.com']
)
.then(function (result){
    console.log('Success')
    connection.client.shutdown()
})
.catch(function (error){
     console.log(error.message)
     connection.client.shutdown()
});